Samples/Sample.Xamarin.Mac/Directory.Build.props

```diff
@@ -0,0 +1,6 @@
+<Project>
```

Comment:
Just curious. I had the same problem on the Unity SDK. Why was this not an issue before?

Reply:
This is a sample that was contributed by an external contributor, it was missing the Directory Build property but at this case there was no real impact to the end user.

Reply:
This file doesn't seem to be needed, did this solve a problem?

Please read on how this works on this page: https://learn.microsoft.com/en-us/visualstudio/msbuild/customize-by-directory?view=vs-2022#directorybuildprops-and-directorybuildtargets